### PR TITLE
Allow K-Line command and regexes to be configured

### PR DESCRIPTION
**Pull request by examknow**
Per #13

### CODE REVIEW

File: config.example.yaml [hunk 1]
```diff
@@ -14,6 +14,11 @@ oper:
   # oper via CHALLENGE key instead
   #file: ~/libera.key
 
+bancmd: "KLINE 1440 {ident}@{user.ip} :{reason}"
```
Comment (jesopo) [inline config.example.yaml:17]: I don't think `{user.ip}` will work here - I think `.format` vars have to be simple key/value

Comment (examknow) [reply config.example.yaml:17]: It seemed to work alright when I tested it

Comment (jesopo) [reply config.example.yaml:17]: perhaps i tested poorly. will try again!

Comment (jesopo) [reply config.example.yaml:17]: well I stand corrected.

```
>>> class A:
...     a=1
...
>>> a = A()
>>> "{a.a}".format(a=a)
'1'
```

**Pull request merged**